source/verify-packages.txt

```diff
@@ -0,0 +1,244 @@
+.. _verify-packages:
+
+==========================================
+Verify Integrity of {+atlas-cli+} Packages
+==========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+The {+atlas-cli+} release team digitally signs all software packages and
+container images to certify that a particular package is valid and
+unaltered. Before installing the {+atlas-cli+}, you should validate the 
+package using the provided PGP signature, SHA-256 checksum, or 
+`Cosign <https://github.com/sigstore/cosign>`__ information.
```

Comment:
Is this required for MacOs? If it applies only to a subset of operating systems, maybe we should call that out.

Reply:
No, it's just for Linux, Windows, and Docker. I'll call that out here